docs/getting_started/adding_a_configuration_file.md

```diff
@@ -5,26 +5,34 @@ You may want to add some settings to your plugin. The best way to do this is thr
 
 ### Creating the configuration file
 
-Create a new class called `MyConfiguration` and add the following decorator to it:
+Create a new class called `MyConfiguration` that extends `Configuration` and add the `@ConfigFile` decorator to it:
 
 ```java
-@ConfigFile(name = "first", type = "json", defaultFile = "defaultFirst")
+@ConfigFile(name = "first", type = "json")

```

Comment:
The `defaultFile` field is actually required on this to tell Conduit where to copy your default configuration from

Reply:
Will Conduit create the file (as an empty file) if that isn't present or will it just not create a file at all?

The reason that I ask is because generally it's better not to include things in the article that aren't related to the instructions at all, and because this guide didn't include anything about setting the default values aside from a footnote (the guide primarily focuses on just reading from the config), it isn't really related and was a redundant optional attribute (afaik), but what I could do is add a "How to create a default config" section for the guide and then that would discuss adding the attribute to the decorator if the user needs it. Which I'll probably do at some point anyway, it just depends on if you want it for this PR.

Though if the attribute is needed for Conduit to create the `PluginName/ConfigName.ConfigType` file, then I'll leave it there and explain why it's there.

Reply:
Technically yes, conduit will try to generate a default based off of the values in the configuration class. So if we wanted to avoid having that parameter for this specific page, you could just add a value to the `joinMesssage` parameter

Reply:
Actually - I'm going to merge this as-is, since I think we could actually make a better guide about configuration defaults separate from this

Reply:
Sounds good to me.